test/test-conditions-1.js

```diff
@@ -0,0 +1,6 @@
+const operations = require("./operations-conditions-1.js");
+const assert = require("assert");
+
+it("Правильно определяет условие conditions-task-1", () => {
```

Comment:
For better test coverage you should have tests for edge cases like:
-`-1` and  `0`
- `0` and `1`
- `1` and `1`

Reply:
edge cases are still not covered